### PR TITLE
Build fix

### PR DESCRIPTION
**Pull request by nithinkrishna**
- Updated frg-runners

### CODE REVIEW

**Comment by nithinkrishna** [comment]
Waiting on https://github.com/ampleforth/frg-ethereum-runners/pull/4

File: package-lock.json [hunk 1]
```diff
@@ -2143,8 +2143,8 @@
       "dev": true
     },
     "frg-ethereum-runners": {
-      "version": "git+https://github.com/ampleforth/frg-ethereum-runners.git#090d0589359a1466c7ad86c64a6a04bc3ef9ac18",
-      "from": "git+https://github.com/ampleforth/frg-ethereum-runners.git#v1.0.0-alpha",
```
Comment (ahnaguib) [inline package-lock.json]: Should we just use master until we launch? and then assign version numbers?

Comment (ahnaguib) [reply package-lock.json]: Just a suggestion feel free to ignore :)

Comment (nithinkrishna) [reply package-lock.json]: will do that if we make any more changes. Hopefully we wont

**Pull request merged**